portfolio/src/main/webapp/script.js

```diff
@@ -35,23 +35,41 @@ async function getComment() {
     if (document.getElementById("comment-container")) {
         console.log("comment container exists")
     } else {
+        document.getElementById("container").style.display = "none";
         //Creates and formats a div that shows the content
-        var d = document.createElement("div");
-        const response = await fetch('/data');
-        const comments = await response.json()
-            .then(comment => {
-                for(const m of comment) {
-                    let para = document.createElement('div');
-                    let message = "\"" + m.content + "\" - " + m.author;
-                    para.appendChild(document.
-                        createElement('strong')).innerHTML = message;
-                    d.appendChild(para);
-                }
-            }
-        )
-        d.className="content";
-        d.id="comment-container"
-        document.body.appendChild(d);
+        var login = await fetch('/login');
+        var result = await login.json();
 
+        //Checks if user is logged in
+        if (result[0].length > 0) {
```

Comment:
If you do decide to make a LoginStatus class for the HomeServlet then this will become `if(result.userEmail){...}` and instead of using `result[1]` below to get the url you can do `result.url`.

Reply:
This is resolved in the most recent commit